pkg/env/env.go

```diff
@@ -58,6 +58,7 @@ const (
 	ServiceKeyvaultSuffix            = "-svc"
 	RPPrivateEndpointPrefix          = "rp-pe-"
 	ProxyHostName                    = "PROXY_HOSTNAME"
+	OIDCBlobContainerPrefix          = "oic-"
```

Comment:
is this already decided on for name?  All our other storage accounts are `-<subject>` rather than `<subject>-`. 

No worries if it's already done.

Reply:
Storage account name is already determined by the STORAGE_ACCOUNT_DOMAIN env variable.
This prefix is used here for creating a container inside the OIDC storage account. But this naming still needs a decision.
Decision here is made from the container naming convention used for bootdiagnostic files in arogwy<region> storage accounts. They have a naming convention as `bootdiagnostics-gatwayvm-<ID>`, also the Private Endpoints do have one-one mapping between cluster and PE and they also uses naming like `rp-pe-<docID>`.